modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -3214,3 +3402,189 @@
 			p.w(format, table.unpack(arg))
 		end
 	end
+
+	--
+	-- Linux project generation functions
+	--
+
+	function m.linuxApplicationType(cfg)
+		m.element("Keyword", nil, "Linux")
+		m.element("RootNamespace", nil, "%s", cfg.project.name)
+		m.element("MinimumVisualStudioVersion", nil, "17.0")
+		m.element("ApplicationType", nil, "Linux")
+		m.element("TargetLinuxPlatform", nil, "Generic")
+		m.element("ApplicationTypeRevision", nil, "1.0")
+	end
+
+	function m.linuxDebugInformationFormat(cfg)
+		if cfg.symbols then
+	
+			if cfg.symbols == p.OFF then
+				m.element("DebugInformationFormat", nil, "None")
+			elseif cfg.symbols == "Full" then
+				m.element("DebugInformationFormat", nil, "FullDebug")
+			else
+				m.element("DebugInformationFormat", nil, "Minimal")
+			end
+
+		end
+	end
+
+	function m.linuxExceptionHandling(cfg, condition)
+		if cfg.exceptionhandling then
+			m.element("ExceptionHandling", condition,  iif(cfg.exceptionhandling == p.OFF, "Disabled", "Enabled"))
+		end
+	end
+
+	function m.linuxFloatingPointModel(cfg)
+
+		if cfg.floatingpoint then
+			m.element("RelaxIEEE", nil, iif(cfg.floatingpoint == p.OFF, "false", "true"))
+		end
+
+	end
+
+	function m.linuxLanguageStandardCpp(cfg)
+		local cpp_langmap = {
```

Comment:
Wonder if we can/should use `gcc.cxxflags.cppdialect` (and remove prefix `--std=`).

Reply:
Where can I find this? I took this code essentially from the Android module

Reply:
I meant that mapping is similar to the one from gcc. Wonder how we can factorize that.

Probably outside of the scope of that PR though.

Reply:
I see, fair enough